.github/workflows/Continuous-integration.yml

```diff
@@ -40,3 +40,27 @@ jobs:
 
       - name: Cypress run
         uses: cypress-io/github-action@v5.3.0
+        
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      
+      - name: Cached LFS checkout
+        uses: nschloe/action-cached-lfs-checkout@v1.1.2
+
+      - name: Run docker-compose up
+        run: docker compose -f docker/docker-compose.yaml up --detach
+
+      - name: Build unit_tests image
+        run: docker build -t unit_tests ./unit_tests/
+      
+      - name: Wait 10 seconds
+        run: sleep 10s
+
+      - name: Get logs
+        run: docker compose -f docker/docker-compose.yaml logs
+        
+      - name: Run Python Unit Tests
+        run: docker run --network rabbitnet unit_tests
```

Comment:
Why is this a separate test? Building all the docker containers 3 times seems like a bit of a waste

Reply:
It is a separate job as it has a separate responsibility from the cypress-tests. 
Furthermore, by having it as a separate job we can parallelize the jobs thus making the overall pipeline run-time faster.
I have, however, changed the step where we build and spin up all the containers to now only do so for the rabbitmq service :)